### ISSUE TITLE
[Custom Threat Intelligence] Add support for client cert authentication

### ISSUE THREAD

**Issue by narph** [elastic/integrations#11585]
Enhancement request: adding support for client cert authentication in the CTI integration

**Comment by chemamartinez** [comment]
@narph SSL options have been added to the integration at https://github.com/elastic/integrations/pull/11867.

I am not 100% sure if changes are required for the TAXII CEL program for any particular TAXII server that could require cert authentication. Do you know if this request comes from any particular use case?

**Comment by narph** [comment]
trying to find the source of the request but no luck so far, @jamiehynds might be able to clarify the requirements here

**Comment by narph** [comment]
@chemamartinez can we close this one?

**Comment by chemamartinez** [comment]
Yes, @narph. Closing this as the SSL options include the support for cert authentication.
